Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,180 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+mysql> SELECT population
+    -> FROM city
+    -> WHERE name="rotterdam";
++------------+
+| population |
++------------+
+|     593321 |
++------------+
+
+3. What's the name of all the countries on the continent ‘North America’ ?
+mysql> SELECT name
+    -> FROM country
+    -> WHERE continent="North America";
++----------------------------------+
+| name                             |
++----------------------------------+
+| Aruba                            |
+| Anguilla                         |
+| Netherlands Antilles             |
+| Antigua and Barbuda              |
+| Bahamas                          |
+| Belize                           |
+| Bermuda                          |
+| Barbados                         |
+| Canada                           |
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+4. What's the top 10 countries by SurfaceArea ?
+mysql> SELECT name
+    -> FROm country ORDER by surfacearea DESC LIMIT 10;
++--------------------+
+| name               |
++--------------------+
+| Russian Federation |
+| Antarctica         |
+| Canada             |
+| China              |
+| United States      |
+| Brazil             |
+| Australia          |
+| India              |
+| Argentina          |
+| Kazakstan          |
++--------------------+
+5. What's the top 10 most populated cities?
+mysql> SELECT name
+    -> FROM city ORDER BY population DESC LIMIT 10;
++-------------------+
+| name              |
++-------------------+
+| Mumbai (Bombay)   |
+| Seoul             |
+| São Paulo         |
+| Shanghai          |
+| Jakarta           |
+| Karachi           |
+| Istanbul          |
+| Ciudad de México  |
+| Moscow            |
+| New York          |
++-------------------+
+6. Select the top 3 countries by population that their names start with the letter ‘P’
+mysql> SELECT name
+    -> FROM country
+    -> WHERE name like "p%" ORDER BY population LIMIT 3;
```

Comment:
You want the 3 biggest countries here, so you should do `ORDER BY population DESC`. It's a good idea to do `SELECT name, population FROM ...` in this query so you can double check that the population really is ordered.